src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/Mutect2.java

```diff
@@ -153,11 +156,19 @@
     @Override
     protected boolean includeReadsWithDeletionsInIsActivePileups() { return true; }
 
+    @Override
+    public boolean useVariantAnnotations() { return true;}
+
     @Override
     public List<ReadFilter> getDefaultReadFilters() {
         return Mutect2Engine.makeStandardMutect2ReadFilters();
     }
 
+    @Override
+    public List<Class<? extends Annotation>> getDefaultAnnotationGroups() {
+        return Mutect2Engine.getStandardMutect2AnnotationGroups();
```

Comment:
Why don't you add an analogous method for the `HaplotypeCallerEngine`, and call it from both `HaplotypeCaller` and `HaplotypeCallerSpark`?